custom-tsdb/values-secondary.yaml

```diff
@@ -2,15 +2,24 @@ kubecostMetrics:
   # Guide: https://docs.kubecost.com/architecture/ksm-metrics#disabling-ksm-emission
   emitKsmV1Metrics: false # emit all KSM metrics in KSM v1 if true.
   emitKsmV1MetricsOnly: true # emit only the KSM metrics missing from KSM v2, this is required to be true.
+kubecostProductConfigs:
+  metricsConfigs:
+    disabledMetrics: # drop additional metrics which are still emitted when "emitKsmV1MetricsOnly"
```

Comment:
Are these metrics that are deprecated in v2?

Reply:
yes, only deprecated and still present.